### ISSUE TITLE
Cuda error 1476 the launch timed out

### ISSUE THREAD

**Issue by matovicv** [KlausT/ccminer#120]
Hello, i get this Cuda error cpu_hash_k4_2stream at line 1476: the launch timed out...
https://imgur.com/a/6PTgQ
Can anyone help. Ty

**Comment by KlausT** [comment]
This looks a little bit wierd.
Are you running two miners at the same time?

**Comment by matovicv** [comment]
No, im using ethcontol application that run ccminer. Tryed with 8.15 and 8.17. 338.13 drivere, Windows 10. I have 1 1060 and 3 1070 cards.  Its Just to monitor ccminer over Internet. Www.ethmonitoring.com

**Comment by lyolyalya** [comment]
in my system i can't use neoscrypt algo with intensity 17.just 16.aaand try to decrease memory overclock.

**Comment by KlausT** [comment]
Are you using Windows?
Could be the 2 second driver timeout caused by the 1060

**Comment by matovicv** [comment]
Windows 10, - i 16.5. I also tryed with out - i. Cards are for 1060 +115 +100 and for 1070 +130 +100. Do you Know how to fix that what you sa Klaust? It was working for 20 days ok and today this error.

**Comment by KlausT** [comment]
intensity 16 should be ok for a 1060
you can set different intensities for your 4 cards, something like `-i 16,17.5,17.5,17.5` if the 1060 is GPU 0
Did you change the overclocking?
Or maybe it's a heat issue

**Comment by matovicv** [comment]
Didnt Know about different intens for every cards, Ty for this. This are oc 1060 +115 +100 and for 1070 +130 +100. Temps are 60-65 degres. Also can you add in next version of KlausT miner Watt per card plssssssssssss Like other ccminer versions have. Ili tryed to instalacija newer versions of drivers. Maybe that help.
